### ISSUE TITLE
Migrate to focus-visible (polyfill)

### ISSUE THREAD

**Issue by DanielRuf** [zurb/foundation-sites#11344]
## What is missing?
Better focus ring support (CSS 4).

## Why is it important?
Better support for focus ring, less dependent thirdparty code.

## Possible Solution
https://github.com/WICG/focus-visible

## Your Environment

- Foundation version(s) used:
- Browser(s) name and version(s):
- Operating System and version (desktop or mobile):

## Checklist (all required):

- [x] I have read and follow the CONTRIBUTING.md document.
- [x] There are no other issues similar to this one.
- [x] The issue title is descriptive.
- [x] The template is correctly filled.

**Comment by Owlbertz** [comment]
Are you planning to remove `what-input` as a dependency? While the spec is not widely implemented this would result in using the polyfill you mentioned so you would exchange one third party library with another?

**Comment by DanielRuf** [comment]
> Are you planning to remove what-input as a dependency?

Yes.

> While the spec is not widely implemented this would result in using the polyfill you mentioned so you would exchange one third party library with another?

It is smaller, by the WICG, just for this and the (CSS) support will gracefully be working in browsers which support it (CSS next and so on).

what-input is a bit misused here for our needs and does much more and does not polyfill any native specification or official specification draft.

**Comment by DanielRuf** [comment]
8kb vs 12kb
2kb min vs 4kb min

**Comment by Owlbertz** [comment]
Ok, I checked and agree that it would be a good idea to use a future standard. But this would most likely be something that must only be done when updating to a major version (i.e. v7) in order to not break anything.

**Comment by SassNinja** [comment]
I like the idea of using native stuff here – definitely more future-proof.

> But this would most likely be something that must only be done when updating to a major version (i.e. v7) in order to not break anything.

Completely agree!
